Astro 10/25 release notes (#1383)
astronomer/docs: astro/release-notes.md

**Latest CLI Version**: 1.6.0 ([Release notes](cli/release-notes.md))

## October 25, 2022 

> ### Simplified Deployment transfers

Reviewer: ```suggestion
### Transfer a Deployment to a different Workspace
```

You couldn't transfer a Deployment before, so I'd just keep it direct here

Author: I tried making the release note titles slightly different from the doc topic so that searches don't confuse them. How about "New Deployment transfer functionality"?

> ### Transfer a Deployment to another Workspace

Author: Can't think of an alternative title that would differentiate this from the actual documentation. @kwcanuck  thoughts?

Reviewer 2: @jwitz - maybe _Transfer Deployments to other Workspaces_ or _Simplified Deployment transfers_?

> ### New Deployment transfer functionality
> 
> You can now transfer a Deployment from one Workspace to another in your Organization. This feature is helpful if you need to change the group of users that have access to a Deployment, or if you create a Deployment in the wrong Workspace. See [Transfer a Deployment to another Workspace](deploy-code.md#transfer-a-deployment-to-another-workspace).
> 

Author: ```suggestion
```

> You can now transfer a Deployment, including all of its DAGs and configurations, to another Workspace on which you have Admin or Editor access. You might want to transfer a Deployment when Deployment ownership moves to a new set of users, or when you create a Deployment in the wrong Workspace. See [Transfer a Deployment to another Workspace](deploy-code.md#transfer-a-deployment-to-another-workspace).

Reviewer: ```suggestion
You can now transfer a Deployment from one Workspace to another in your Organization. This feature is helpful if you need to change the group of users that have access to a Deployment, or if you create a Deployment in the wrong Workspace. You must be a Workspace Admin in the original Workspace and an Admin or Editor in the target Workspace to transfer a Deployment.

See [Transfer a Deployment to another Workspace](deploy-code.md#transfer-a-deployment-to-another-workspace).
```

> You can now transfer a Deployment, including all of its DAGs and configurations, to another Workspace for which you have Admin or Editor access. You might want to transfer a Deployment if the purpose of a Deployment has changed such that it belongs to a new set of users within your Organization, or if you created a Deployment in the wrong Workspace by mistake. See [Transfer a Deployment to another Workspace](deploy-code.md#transfer-a-deployment-to-another-workspace).

Reviewer 2: ```suggestion
You can now transfer a Deployment, including all of its DAGs and configurations, to another Workspace on which you have Admin or Editor access. You might want to transfer a Deployment when Deployment ownership moves to a new set of users, or when you create a Deployment in the wrong Workspace. See [Transfer a Deployment to another Workspace](deploy-code.md#transfer-a-deployment-to-another-workspace).
```
